processing/app/models.py

```diff
@@ -1,6 +1,24 @@
+import uuid
+from datetime import datetime
+from typing import Optional
 from pydantic import BaseModel
+from enum import Enum
+from sqlmodel import Field, SQLModel
 
 class Item(BaseModel):
     name: str
     price: float
     is_offer: bool = False
+
+class StatusEnum(str, Enum):
```

Comment:
As of python 3.12 [StrEnum](https://docs.python.org/3/library/enum.html#enum.StrEnum) was introduced. Use it instead of `StatusEnum(str, Enum)`